### DISCUSSION TITLE
separate PRs for major version update of jest and ts-jest

### DISCUSSION THREAD

**Discussion by travi** [renovatebot/renovate#10698] [Request Help]
i added renovate to a private monorepo this week where a package depends on both jest and ts-jest. the recent major versions of those two dependencies need to be updated together, but i interpreted the [addition of the `jestPlusTSJest` group and enabling it by default](https://github.com/renovatebot/renovate/pull/10596) was meant to handle this situation by upgrading both in the same PR. however, these two packages received update PRs independently.

to verify that the behavior was not limited to that private repo, i added a new package to an existing public repo that follows a similar structure. the behavior was the same and you can see the separate PRs [here for jest](https://github.com/travi-test/lerna-playground/pull/276) and [here for ts-jest](https://github.com/travi-test/lerna-playground/pull/277). in this case the automerge did work because the new package wasn't actually using the two packages in a way that required them being updated together, but verification failed in the original repo and prevented merging.

am i misunderstanding how that new group should behave?

**Comment by rarkins** [comment]
This preset is not default Renovate behaviour but instead default "config:base" preset behaviour. Are you using it? If not then you can add just that preset alone if you prefer.

**Reply by travi** [reply]
sorry, you're right. i should have been more clear. yes, both projects extend `config:base`.

the [public reproduction's config](https://github.com/travi-test/lerna-playground/blob/master/.renovaterc.json) extends [this shareable config](https://github.com/form8ion/renovate-config/blob/master/default.json), where [`config:base` is extended](https://github.com/form8ion/renovate-config/blob/35d994f9572feb6643cb222292165ea5254455cc/default.json#L2)

**Reply by rarkins** [reply]
Can you file it as a bug then with minimal reproduction? Ie just 2 deps for easier debugging.

**Reply by travi** [reply]
i'll get a bug filed soon, but in the meantime, here are the separate PRs for a minimal repo:
* [jest](https://github.com/travi-test/jest-ts-reproduction/pull/2)
* [ts-jest](https://github.com/travi-test/jest-ts-reproduction/pull/3)

**Reply by travi** [reply]
https://github.com/renovatebot/renovate/issues/10702
